ci/ci/constants.py

```diff
@@ -22,5 +22,6 @@
     'daniel-goldstein',
     'ahiduchick',
     'GreatBrando',
-    'johnc1231'
+    'johnc1231',
```

Comment:
you'll have to rebase anyhow once Milo's change goes in (he's editing this same location) so you can either wait until that goes in and rebase, or split this out into a separate PR and we can get this one in now

Reply:
Will split out.